docs/developing/developing_secrets.md

```diff
@@ -0,0 +1,47 @@
+---
+
+copyright:
+years: 2021
+lastupdated: "2021-08-16"
+
+---
+
+{:new_window: target="blank"}
+{:shortdesc: .shortdesc}
+{:screen: .screen}
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:child: .link .ulchildlink}
+{:childlinks: .ullinks}
+
+# Hello world with a secret
+{: #secrets}
+
+This example helps you learn how to develop an {{site.data.keyword.edge_service}} that uses secrets. You use secrets to ensure that login credentials and other sensitive information are kept secure
+{:shortdesc}
+
+## Before you begin
+{: #secrets_begin}
+
+Complete the prerequisite steps in [Preparing to create an edge service](service_containers.md). As a result, these environment variables should be set, these commands should be installed, and these files should exist:
```

Comment:
Should we offer any info about what to do IF the files don't exist?

Reply:
No, because they will exist if you followed the instructions in the referenced section. This text is copied from the other developing examples.